### DISCUSSION TITLE
Form elements with modern CSS features

### DISCUSSION THREAD

**Discussion by sergey-dme** [Budibase/budibase#2011] [Ideas]
Hi all, 

It would be great to have in builder form elements/components with modern CSS features, especially the checkbox/switch.

[The examples with code shown here](https://css-tricks.com/custom-styling-form-inputs-with-modern-css-features/)

Hope this will be part of Theming https://github.com/Budibase/budibase/issues/1847 

Thanks.

**Comment by joebudi** [comment]
Hey!

This will not be part of theming.
